auto_install/install.sh

```diff
@@ -21,7 +21,7 @@ PKG_CACHE="/var/lib/apt/lists/"
 UPDATE_PKG_CACHE="${PKG_MANAGER} update"
 PKG_INSTALL="${PKG_MANAGER} --yes --no-install-recommends install"
 PKG_COUNT="${PKG_MANAGER} -s -o Debug::NoLocking=true upgrade | grep -c ^Inst || true"
-PIVPN_DEPS=(openvpn git tar wget grep iptables-persistent dnsutils expect whiptail net-tools)
+PIVPN_DEPS=(openvpn git tar wget grep iptables-persistent dnsutils expect whiptail net-tools uuidgen)
```

Comment:
@IcedComputer 
`uuidgen` is no package on Debian/Raspbian. It is part of the [uuid-runtime](https://packages.debian.org/buster/amd64/uuid-runtime/filelist) package.

The essential [util-linux](https://packages.debian.org/buster/util-linux) depends on [libuuid1](https://packages.debian.org/buster/libuuid1), which recommends but not depends on `uuid-runtime`. So to be failsafe it indeed should be installed.

But, why not simply read `/proc/sys/kernel/random/uuid` instead of adding another dependency? Is this not present on all systems? This is the method we use since years across nearly all ARMs on the marked, x86_64 PCs and VMs and it works on all of them (with Debian/Raspbian):
```
NEW_UUID=$(</proc/sys/kernel/random/uuid)
```

Reply:
damn!! Great catch!! Thanks a lot for that!